PolyclinicLab/Polyclinic.Domain/Interfaces/IRepository.cs

```diff
@@ -0,0 +1,25 @@
+﻿namespace Polyclinic.Domain.Interfaces;
+
+/// <summary>
+/// Generic repository interface defining basic CRUD operations 
+/// for working with entities of type <typeparamref name="T"/>.
+/// </summary>
+/// <typeparam name="T">Type of the entity.</typeparam>
+/// <typeparam name="TId">Type of the entity identifier.</typeparam>
+public interface IRepository<T, TId>
+{
+    /// <summary>Creates a new entity and returns the created entity with its assigned identifier.</summary>
+    T Create(T entity);
```

Comment:
Во всех интерфейсах должны присутствовать модификаторы доступа.

```suggestion
    public T Create(T entity);
```